header.php

```diff
@@ -160,11 +161,11 @@
 							<div class="span10">
 							    <div class="input-append pull-right global-search hidden-phone">
                                 
-                               		 <form id="search" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
+                               		 <form id="search" role="search" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>">
                                         <span aria-hidden="true" data-icon="&#xf002;"></span>
-                                        <input class="span3" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
-										<input type='hidden' name='post_type' value='programs,segments,staff,events,ads' />
-                                         <input id="searchsubmit" value="Search" type="submit" class="btn" />
+                                        <input aria-label="Search" class="span3" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>"/>
+										<input aria-hidden="true", type='hidden' name='post_type' value='programs,segments,staff,events,ads' />
```

Comment:
This isn't needed- hidden fields are already hidden from screen readers

Reply:
This code block becomes unhidden when the menu is activated. See `unhide-nav.js`

Reply:
@andrewpking  - this still needs to be resolved- at the very least the comma needs to be removed between the `aria-hidden` attribute and the `type` attribute. I would recommend fully removing the `aria-hidden`, however